src/App.tsx

```diff
@@ -15,71 +17,75 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
```

Comment:
Change the initial value of 'selectedGood' to an empty string, so no good is selected by default.

Reply:
didn't fix it